Mathlib/Topology/Algebra/Valued/ValuativeRel.lean

```diff
@@ -144,12 +180,18 @@ variable {R : Type*} [CommRing R] [ValuativeRel R] [UniformSpace R]
 
 /-- Helper `Valued` instance when `ValuativeTopology R` over a `UniformSpace R`,
 for use in porting files from `Valued` to `ValuativeRel`. -/
-scoped instance : Valued R (ValuativeRel.ValueGroupWithZero R) where
+instance : Valued R (ValuativeRel.ValueGroupWithZero R) where
   v := ValuativeRel.valuation R
-  is_topological_valuation := IsValuativeTopology.mem_nhds_iff
+  is_topological_valuation := IsValuativeTopology.mem_nhds_zero_iff
 
 end Valued
 
+instance (R : Type*) [CommRing R] [ValuativeRel R] [TopologicalSpace R] [IsValuativeTopology R] :
```

Comment:
should this (and the other instances in this file) get a lower priority?

Reply:
What should the number be?

Reply:
done, and I've also shuffled around their positions